front_end/core/root/Runtime.ts

```diff
@@ -515,7 +513,6 @@ export type Condition = (config?: HostConfig) => boolean;
 export const conditions = {
   canDock: () => Boolean(Runtime.queryParam('can_dock')),
   notSourcesHideAddFolder: () => Boolean(Runtime.queryParam(ConditionName.NOT_SOURCES_HIDE_ADD_FOLDER)),
-  reactNativeUnstableNetworkPanel: () =>
-      Boolean(Runtime.queryParam(ConditionName.REACT_NATIVE_UNSTABLE_NETWORK_PANEL)) ||
-      experiments.isEnabled(ExperimentName.ENABLE_NETWORK_PANEL),
+  reactNativeUnstableNetworkPanel: () => true,
```

Comment:
Why do we still need this flag here? Can we remove it?

Reply:
For sure. Missed this.